release_creation/main.py

```diff
@@ -38,24 +40,37 @@ def main():
     args = parser.parse_args()
     version = args.input_version
     operation = args.operation
-    latest_version, is_draft, latest_release = get_latest_bundle_release(version)
-    logger.info(f"Retrieved latest version: {latest_version} "
-                f"and latest release: {latest_release.tag_name if latest_release else None}")
+    if version.startswith("0.0"):
+        latest_version = Version(DEV_VERSION)
+        is_draft = False
+        latest_release = None
```

Comment:
Won't this fail when there are multiple dev versions with the same tag?  Or will the create action just overwrite the previous release?  I would expect an error.

Reply:
the create action overwrites the previous release